qualtran/resource_counting/__init__.py

```diff
@@ -31,4 +31,6 @@
 
 from ._costing import GeneralizerT, get_cost_value, get_cost_cache, query_costs, CostKey, CostValT
 
+from ._qubit_counts import QubitCount
```

Comment:
why is it a private module?

Reply:
This is to force imports of the form
```
from qualtran.resource_counting import QubitCount
```

instead of

```
from qualtran.resource_counting.qubit_counts import QubitCount
```

Reply:
the underscore isn't necessary though right? You could still do `from .qubit_counts import QubitCount` no? Or are you saying you don't want me to import from qubit_counts ever?

Reply:
the latter